db_interactions.py

```diff
@@ -19,14 +19,14 @@ def insert(query, params=()):
     return "Error inserting data", 500
 
 def select(query, params=()):
-  try:
+  # try:
```

Comment:
This try catch might be important for other endpoints

Reply:
oops forgot i commented that out my bad